tests/jobs/test_backfill_job.py

```diff
@@ -290,13 +306,13 @@ def test_backfill_conf(self):
 
         dr = DagRun.find(dag_id='test_backfill_conf')
 
-        assert conf_ == dr[0].conf
+        assert conf_ == dr[1].conf
```

Comment:
Previously, the get_dummy_dag doesn't create dagrun. Now it creates dag_run, so when the  BackFillJob runs, it creates extra two dagruns which now have the conf. Escaping the first dagrun created by get_dummy_dag_and_run fixture, we now find the conf of the 2nd dagrun=>dr[1]

Reply:
Would it be easier to pass additional arguments to `find` to get only that second run? (e.g. `run_id`)